.changelog/2130.internal.md

```diff
@@ -0,0 +1,11 @@
+client protocol: Add IsReady() and WaitReady() RPC methods
+
+Beside `IsSynced()` and `WaitSynced()` which are triggered when a node is
+registered, new `IsReady()` and `WaitReady()` methods have been added to
```

Comment:
The `*Synced` methods are not triggered when a node is registered, only when the consensus backend is synced. Only the new `*Ready` calls include node registration.